main.py

```diff
@@ -0,0 +1,31 @@
+import win32api
+import win32process
+import win32gui
+import argparse
+
+
+def build_parser():
+    parser = argparse.ArgumentParser(description='This is a title changer')
+    parser.add_argument('pid', action="store", type=int)
+    parser.add_argument('new_title', action="store", type=str)
+
+    parse_results = parser.parse_args()
+
+    return parse_results
+
+
+def find_it(hwnd, parser):
+    ttl = win32gui.GetWindowText(hwnd)
+    t, p = win32process.GetWindowThreadProcessId(hwnd)
```

Comment:
If you're calling a function and not using one of the variables, you can ignore them like this:
```python
_, t = win32.process.Get...
```